src/components/CardApplicantList/CardApplicantList.jsx

```diff
@@ -1,70 +1,113 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import { Button, Box, Typography } from '@mui/material';
+import Card from '@mui/material/Card';

```

Comment:
Card is a component that could stay with the other components 
```jsx
import { Button, Box, Typography, Card } from '@mui/material';
```

I say this only to mantain the standard